test/EIP7702Proxy/coinbaseImplementation.t.sol

```diff
@@ -149,15 +149,8 @@ contract CoinbaseImplementationTest is Test {
      * @param target The address where the proxy code should be etched
      */
     function _deployProxy(address target) internal {
-        // Deploy proxy normally first to get the correct immutable values
-        EIP7702Proxy newProxy = new EIP7702Proxy(
-            address(implementation),
-            initSelector,
-            nonceTracker
-        );
```

Comment:
notice there was no need for an additional proxy template to reference when creating certain test situations